client/modules/components/src/Wikitext.svelte

```diff
@@ -0,0 +1,147 @@
+<!--
+  @component Generic wikitext container.
+-->
```

Comment:
In this circumstance yes, but there may be other uses for this component which is why there is so many input options. I have no idea what those scenarios are because we haven't discovered them yet.